yb-voyager/src/queryparser/query_parser.go

```diff
@@ -72,10 +112,22 @@ func GetObjectTypeAndObjectName(parseTree *pg_query.ParseResult) (string, string
 		}
 		funcNameList := stmt.GetFuncname()
 		return objectType, getFunctionObjectName(funcNameList)
+	case isViewStmt:
+		viewName := viewNode.ViewStmt.View
+		return "VIEW", getObjectNameFromRangeVar(viewName)
+	case IsMviewObject(parseTree):
+		intoMview := createAsNode.CreateTableAsStmt.Into.Rel
+		return "MVIEW", getObjectNameFromRangeVar(intoMview)
 	}
 	return "", ""
```

Comment:
let's throw an error if we don't know how to handle it for the statement type? Blindly returning empty strings is a bit error-prone. 

As a result, let's call this only in case of mview/view/plpgsql if-else branches in GetIssues.